services/github/github-stars.tester.js

```diff
@@ -20,3 +20,17 @@ t.create('Stars (repo not found)')
     label: 'stars',
     message: 'repo not found',
   })
+
+// https://github.com/badges/shields/issues/4982
+// It doesn't seem important that this work with trailing spaces, though if it
+// did we'd probably want a more global fix.
+t.create('Stars (repo name with trailing spaces)')
+  .get('/badges/shields     .json')
```

Comment:
> It doesn't seem important that this work with trailing spaces, though if it did we'd probably want a more global fix.

Yeah I can't imagine GH allows trailing spaces in the repo name

Reply:
Weirdly the API accepts them! And then we crash in rendering.